include/maliput/test_utilities/mock.h

```diff
@@ -31,6 +31,7 @@
 
 #include <memory>
 #include <string>
+#include <vector>
```

Comment:
I don't think this change is intentional

Reply:
It is. It was added since std::vector is used in the mocked API.